### DISCUSSION TITLE
What happens to custom ingest parsers on SOUP

### DISCUSSION THREAD

**Discussion by stijnpieters** [Security-Onion-Solutions/securityonion#3186] [Unsupported Versions]
SO2.3.21
1 managersearch node
2 forward nodes

I know custom parsers need to be placed in _/opt/so/saltstack/local/salt/elasticsearch/files/ingest/_, but to get to these parsers I had to make some changes to _/opt/so/saltstack/local/salt/elasticsearch/files/ingest/_.

Do these changes get overwritten if I were to upgrade SO to a new version and do I put them back manually, or can this last file be put in a .../local/salt/... directory as well?

**Comment by weslambert** [comment]
Files/changes in `/opt/so/saltstack/local/` do not get overwritten during updates.
